project-management-infrastructure/src/main/java/life/qbic/projectmanagement/infrastructure/sample/openbis/OpenbisConnector.java

```diff
@@ -314,10 +315,7 @@ public void updateAll(
       Collection<life.qbic.projectmanagement.domain.model.sample.Sample> samples)
       throws SampleNotUpdatedException {
     try {
-      /*FixMe Throws org.springframework.remoting.RemoteAccessException: Could not access HTTP invoker remote service
-          and invalid stream header: 3C68746D
-      */
-      // updateOpenbisSamples(convertSamplesToSampleUpdates(samples));
+      updateOpenbisSamples(convertSamplesToSampleUpdates(samples));
```

Comment:
as it is `ch.ethz.sis.openbis.generic.asapi.v3.dto.sample.update.UpdateSamplesOperation` that expects a mutable list, maybe add a comment to `updateOpenbisSamples` and ensure that it is a mutable list in `updateOpenbisSample` as well. This does remove the need of having an modifiable `SampleUpdate`s list.